dashboard/app/models/levels/level.rb

```diff
@@ -799,6 +803,18 @@ def summarize_for_lesson_show(can_view_teacher_markdown)
     }
   end
 
+  # Summarize the properties for a lab2 level.
+  # Called by ScriptLevelsController.level_properties.
+  # These properties are usually just the serialized properties for
+  # the level, which usually include levelData.  If this level is a
+  # StandaloneVideo then we put its properties into levelData.
+  def summarize_for_lab2_properties
+    video = specified_autoplay_video&.summarize(false)&.camelize_keys
```

Comment:
would it be possible here to add a `type` property (ex. for Music is would be `'music'`, for Video it would be `'video'`? This may make our lives easier on the frontend given we are using a [union type](https://www.typescriptlang.org/docs/handbook/unions-and-intersections.html#discriminating-unions), and will be adding to it going forward.

Reply:
I can add it here, though I'm not sure what to do with it on the front-end apart from double-checking that it matches expectations and maybe logging some kind of error if it isn't?

Reply:
It will help with type checking--if we know something has a `music` type we then will know which other fields it has. But it's not strictly necessary, it just could give us cleaner code down the line.

Reply:
Sure thing.  As a note, I used the `type` property from `Level`, which comes through as `"Music"` or `"StandaloneVideo"`.